compiler/p/codegen/PPCSystemLinkage.cpp

```diff
@@ -1439,10 +1439,13 @@ void TR::PPCSystemLinkage::buildDirectCall(TR::Node *callNode,
             {
             //For Little Endian, load target's Global Entry Point into r12, TOC will be restored at branch target.
             TR::Register *geReg = dependencies->searchPreConditionRegister(TR::RealRegister::gr12);
-            generateTrg1MemInstruction( cg(),TR::InstOpCode::Op_load, callNode, geReg,
+            if (!cg()->comp()->getOption(TR_DisableTOC))
+               generateTrg1MemInstruction(cg(),TR::InstOpCode::Op_load, callNode, geReg,
                   new (cg()->trHeapMemory()) TR::MemoryReference(cg()->getTOCBaseRegister(),
```

Comment:
isn't this a reference tot "TOCBaseRegister" ?

Reply:
that is where it is fixed exactly.  notice that reference is under  !disableTOC  condition.

Reply:
Ok, yes I understand the point of the commit better now. For future reference, please let the reviewer resolve the conversation rather than answering the question and then marking it resolved yourself.